typescript/framework-extensions/vercel-ai-sdk/package.json

```diff
@@ -41,8 +41,10 @@
     "ai-sdk"
   ],
   "dependencies": {
-    "@coinbase/agentkit": "^0.2.3",
-    "ai": "^4.1.16",
     "zod": "^3.22.4"
+  },
+  "peerDependencies": {
+    "@coinbase/agentkit": ">=0.1.0",
```

Comment:
Using `>=` so that pre-v1 minor bumps are included. If we were to use caret `^`, minor versions above the specified wouldn't match. For example, if we were to use `^0.2.3`, consumers would be forced to use a `0.2.x` version, but any `0.3.x` or higher would be invalid. This will be the case until we reach `v1`, at which point we can switch back to using caret.

Reply:
Should we document anywhere that this pattern should be changed once we're at v1? I know we're a ways away, but it seems like a very easy thing to miss once we do hit that milestone

Reply:
Yep, I'll track it in a ticket

Reply:
https://linear.app/coinbase/issue/AGENT-257/update-peerdep-on-agentkit-core